### PR TITLE
Feat/207 profile page

### PR DESCRIPTION
**Pull request by joonatanaatos**
## WHY:

Fixes [Issue link]

## Check off each of the following tasks as they are completed

- [x] I have reviewed the changes myself in this PR. Please check the [self-review document](https://github.com/OpenVAA/voting-advice-application/blob/main/docs/contributing/self-review.md)
- [x] I have added or edited unit tests.
- [x] I have run the unit tests successfully.
- [x] I have tested this change on my own device.
- [ ] I have tested this change on other devices (Using Browserstack is recommended).
- [x] I have tested my changes using the [WAVE extension](https://wave.webaim.org/extension/)
- [x] I have added documentation where necessary.
- [x] Is there an existing issue linked to this PR?

**Clean up your git commit history before submitting the pull request!**

### CODE REVIEW

File: frontend/src/lib/i18n/translations/fi/candidateApp.json [hunk 1]
```diff
@@ -1,152 +1,156 @@
...
-    "repetition": "FI - Contains character repetition",
-    "commonPatterns": "FI - Contains common sequences or patterns"
+    "length": "Vähintään {minPasswordLength} merkkiä",
+    "uppercase": "Iso kirjain",
+    "lowercase": "Pieni kirjain",
+    "number": "Numero",
+    "symbol": "Symboli",
+    "username": "Vältä nimesi käyttämistä salasanassa",
+    "repetition": "Sisältää merkkien toistoa",
+    "commonPatterns": "Sisältää yleisiä sekvenssejä tai kuvioita"
```
Comment (Henriki2305) [inline frontend/src/lib/i18n/translations/fi/candidateApp.json:92]: "sekvenssejä tai kuvioita" -> "merkkisarjoja"

File: frontend/src/lib/i18n/translations/fi/candidateApp.json [hunk 2]
```diff
@@ -1,152 +1,156 @@
...
-    "revealPassword": "FI - Reveal password"
+    "hidePassword": "Piilota salasana",
+    "revealPassword": "Näytä salasana"
   },
   "homePage": {
-    "greeting": "FI - Hello, {userName}!",
-    "explanation": "FI - In order to appear in the Election Compass, you have to enter some basic information about yourself as well as answer opinion statements concerning political issues.",
-    "buttonText": "FI - Continue"
+    "greeting": "Hei, {userName}!",
+    "explanation": "Voidaksesi esiintyä vaalikompassissa sinun on annettava joitakin perustietoja itsestäsi ja vastattava poliittisia kysymyksiä koskeviin mielipidekysymyksiin.",
```
Comment (Henriki2305) [inline frontend/src/lib/i18n/translations/fi/candidateApp.json:149]: "vastattava poliittisia kysymyksiä koskeviin mielipidekysymyksiin" -> "ottaa kantaa poliittisiin kysymyksiin"

**Review by Henriki2305 — APPROVED** [review]
Fixed some mistakes and proposed more formal sentences/terms as suggestions

File: frontend/src/lib/i18n/translations/fi/candidateApp.json (hunk 2 above)
Comment (joonatanaatos) [reply frontend/src/lib/i18n/translations/fi/candidateApp.json:149]: I'll put "vastattava poliittisiin mielipidekysymyksiin"

File: frontend/src/lib/i18n/translations/fi/candidateApp.json [hunk 3]
```diff
@@ -1,152 +1,156 @@
...
   "setPassword": {
-    "greeting": "FI - Hello, {userName}!",
-    "description": "FI - Set a secure password according to the requirements below:",
-    "password": "FI - Password",
-    "confirmPassword": "FI - Confirm password",
-    "setPassword": "FI - Set password",
-    "passwordsDontMatch": "FI - Passwords don't match",
-    "passwordNotValid": "FI - Password is not valid",
-    "registrationError": "FI - Registration failed"
+    "greeting": "Terve {username}!",
```
[the remark this answers is not in the document]
Comment (joonatanaatos) [reply frontend/src/lib/i18n/translations/fi/candidateApp.json:75]: In finnish, there is no comma there

File: frontend/src/lib/i18n/translations/fi/candidateApp.json [hunk 4]
```diff
@@ -1,152 +1,156 @@
...
-    "errorMessage": "FI - An error occurred when sending data!",
-    "pending": "FI - pending",
-    "addAnother": "FI - Add another",
-    "selectFirst": "FI - Select first",
-    "unaffiliatedDescription": "FI - Set this on if you're not a member of the party on whose list you are.",
-    "nominationsDescription": "FI - These are the constituencies where you're nominated along with the election list and possible election symbol. If a nomination is marked 'Pending', your election symbol might not yet be defined.",
-    "electionManifesto": "FI - Election manifesto",
-    "nominations": "FI - Nominations",
-    "tapToAddPhoto": "FI - Tap to add photo"
+    "errorMessage": "Tapahtui virhe tietoja lähetettäessä!",
```
[the remark this answers is not in the document]
Comment (joonatanaatos) [reply frontend/src/lib/i18n/translations/fi/candidateApp.json:43]: I think the passive form of the verb is more formal in this case, so I'll keep it the same but change the word order a bit to make it sound more natural

File: frontend/src/hooks.server.ts
Comment (norppaa) [inline frontend/src/hooks.server.ts:1]: Consider fixing this

**Review by norppaa — CHANGES_REQUESTED** [review]
lgtm

**Review by norppaa — APPROVED** [review]

**Pull request merged**